station/app/schemas/local_trains.py

```diff
@@ -27,25 +27,87 @@ class LocalTrain(DBSchema):
 
 
 class LocalTrainCreate(BaseModel):
-    train_name: str
+    train_name: Optional[str] = None
 
+class LocalTrainConfigBase(BaseModel):
+    name: str
+    image: Optional[str] = None
+    tag: Optional[str] = None
+    query: Optional[str] = None
+    entrypoint: Optional[str] = None
+    volumes: Optional[str] = None
+    env: Optional[str] = None
+    train_id: Optional[str] = None
 
-class LocalTrainAddMasterImage(BaseModel):
-    train_id: str
-    image: str
 
-class LocalTrainGetFile(BaseModel):
-    train_id: str
-    file_name: str
+class LocalTrainConfigCreate(LocalTrainConfigBase):
+    pass
+
 
-class LocalTrainAddTag(BaseModel):
+class LocalTrainConfigUpdate(LocalTrainConfigBase):
+    pass
+
+class LocalTrainAirflowConfig(BaseModel):
+    image: Optional[str] = None
+    repository: Optional[str] = None
+    tag: Optional[str] = None
+    env: Optional[str] = None
+    query: Optional[str] = None
+    entrypoint: Optional[str] = None
+    volumes: Optional[str] = None
+    train_id: Optional[str] = None
+
+
+class LocalTrainInfo(BaseModel):
+    id: str
     train_id: str
-    tag: str
+    train_name: str
+    config_id:  Optional[str]
+    airflow_config_json: Optional[LocalTrainAirflowConfig]
+    created_at: datetime
+    updated_at: Optional[datetime]
+    is_active: bool
+
+
+class LocalTrainSchemaList(BaseModel):
+    trains: list[LocalTrainInfo]
```

Comment:
return list of LocalTrainInfo directly from route instead of initializing another object